homeassistant/components/zha/select.py

```diff
@@ -679,3 +684,133 @@ class KeypadLockout(ZCLEnumSelectEntity):
     _enum = KeypadLockoutEnum
     _attr_translation_key: str = "keypad_lockout"
     _attr_icon: str = "mdi:lock"
+
+
+class DanfossExerciseDayOfTheWeekEnum(types.enum8):
+    """Day of the week."""
+
+    Sunday = 0
+    Monday = 1
+    Tuesday = 2
+    Wednesday = 3
+    Thursday = 4
+    Friday = 5
+    Saturday = 6
+    Undefined = 7
```

Comment:
This enum is already defined in zha-quirks, so we can just use that too.
(I know some custom entities do it differently, but I think we changed to directly using the quirks enum if possible(?))

Reply:
I'll do that everywhere. It is just difficult to test right now, because my quirks PR isn't merged yet and I have to hack my code constantly to be able to test it easily on my system.